docs/source/release_notes.rst

```diff
@@ -2,6 +2,14 @@ Release Notes
 -------------
 
 **Future Releases**
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+
+**v0.14.1 Sep. 29, 2020**
```

Comment:
Less that 2 weeks have passed since 0.13.2 and we've already got over 30 merges! 🔥🔥🔥